.github/workflows/release_asset.yml

```diff
@@ -0,0 +1,51 @@
+name: Release asset bot
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+jobs:
+  build:
+    name: Build release asset
+    continue-on-error: true
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - ubuntu-latest
+          - ubuntu-18.04
+          - macos-latest
+          - windows-latest
```

Comment:
We do want move to work properly on windows as well as these other platforms, do we currently have a CI job that actually builds and runs tests on all of these platforms?

Reply:
I'm not sure, but the move-analyzer VSCode plugin requires a Windows version binary.

Reply:
> do we currently have a CI job that actually builds and runs tests on all of these platforms?

AFAIK ubuntu is the ONLY platform on which we run our CI tests. I guess it would be tricky to have a mac os or windows based testing platform even if this is something we should ideally have.

Reply:
I can add some CI tests, on mac and windows platforms